waku/v2/node/wakunode2.nim

```diff
@@ -136,7 +141,10 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   ##
   let
     rng = crypto.newRng()
-    hostAddress = tcpEndPoint(bindIp, bindPort)
+    hostAddress = tcpEndPoint(bindIp, bindPort) 
+
+    hostAddressWithWss = hostAddress & addWssFlag.get()
```

Comment:
I don't think this is used anywhere? It should be possible to build a `switch` with both TCP and wss endpoints (use `withAddresses()` rather than `withAddress()`). Also, your host would need to use a different TCP port for the wss endpoint.

Reply:
I'd expect the host to be able to run both TCP and WSS at the same time (just in case it was not clear).

Reply:
I was reading about the same, as per libp2p documentation this case of multiple transport is handled using circuit relay, however I am not sure, I am trying to get more information, and to check if there is some workaround.